docker-compose.override.yml

```diff
@@ -37,3 +37,16 @@ services:
         "-c",
         "wait-for-it.sh db:5432 -- ./manage.py migrate && ./manage.py runserver 0.0.0.0:8000",
       ]
+
+  ember:
+    image: danlynn/ember-cli:latest
+    volumes:
+      - ./ember:/myapp
```

Comment:
Nitpick: i don't really like the naming of `myapp` i would prefer `app` a lot more. `myapp` sounds very templaty and unfinished. Do you know the reason we use `myapp`?

Reply:
It's given by [danlynn/ember-cli](https://hub.docker.com/r/danlynn/ember-cli/).

Reply:
I think they intended it as a template, so do you think we should change it and if yes to `app` or to `mySAGW`?

Reply:
> This image contains everything you need to have a working development environment for ember-cli. The container's working dir is /myapp so that you can setup a volume mapping your project dir to /myapp in the container

We would need to use a custom Dockerfile, just to change the working dir in the container. Is this really worth it?